client/styles/components/_toast.scss

```diff
@@ -3,8 +3,10 @@
   color: $toast-text-color;
   padding: #{10 / $base-font-size}rem;
   position: absolute;
-  top: 0;
-  right: 40%;
+  bottom: 5vh;
```

Comment:
could you use consistent measurements here? for the site i've been using `rem`s, which are easy to convert with SCSS:
`#{<pixels> / $base-font-size}rem;`